samples/requirements.txt

```diff
@@ -2,3 +2,4 @@
 optimum[openvino]==1.20.0
 einops==0.8.0  # For Qwen
 transformers_stream_generator==0.0.5  # For Qwen
+-r ../image_generation/requirements.txt
```

Comment:
Reminder to inverse the dependency or introduce a common requirements.txt

Reply:
Fixed